Documentation/contributing/development/dev_setup.rst

```diff
@@ -190,14 +190,17 @@ and ports to enable NFS.
 .. note::
 
    Make sure your host NFS configuration is setup to use tcp:
-   # cat /etc/nfs.conf
-   ...
-   [nfsd]
-   # grace-time=90
-   tcp=y
-   # vers2=n
-   # vers3=y
-   ...
+
+   .. code-block:: none
```

Comment:
You might consider using `shell-session` highlights:

```suggestion
   .. code-block:: shell-session
```

Reply:
I tried that first, but that made the `cat` output also highlighted (i.e. the comments of the file were grayed-out), so decided to go for `none`.